src/finn/custom_op/quantavgpool2d.py

```diff
@@ -17,19 +18,48 @@ def get_nodeattr_types(self):
             "ibits": ("i", True, 1),
             "obits": ("i", True, 1),
             "signed": ("i", True, 0),
+            "data_layout": ("s", False, "NCHW"),
```

Comment:
add comments to attributes and specify what the allowed values are

Reply:
also, please add specific support for this node/attribute to:
https://github.com/Xilinx/finn/blob/f525398cbc3bb1a6b83be51b11e7923ed7c8870a/src/finn/transformation/infer_data_layouts.py#L41